macro/Cargo.toml

```diff
@@ -27,10 +27,10 @@ quote = "1.0.8"
 syn = "1.0.58"
 scale-info = "1.0.0"
 
-subxt-codegen = { path = "../codegen", version = "0.16.0" }
+subxt-codegen = { path = "../codegen", version = "0.17.0" }
 
 [dev-dependencies]
 pretty_assertions = "1.0.0"
-subxt = { path = "../subxt", version = "0.16.0" }
```

Comment:
Let's remove `version` from this; if I read it right, doing so may mean we no logner need to use `cargo hack` to publish this crate as `cargo publish` will strip the dependency automatically!